src/widget/sidebar.c

```diff
@@ -332,27 +333,49 @@ static void draw_extra_info_objective(int x_offset, int y_offset, int text_group
 static void draw_extra_info_panel(int x_offset, int extra_info_height)
 {
     int y_offset = FILLER_Y_OFFSET;
-
+    int y_current_line = y_offset;
+    // Space between lines, when line is purely text (no buttons)
+    int y_line_space = EXTRA_INFO_HEIGHT_LINE_SPACE;
```

Comment:
Assigning a constant value to `y_line_space` seems needless TBH. Just use `EXTRA_INFO_HEIGHT_LINE_SPACE` instead.

Reply:
the name is so long :smile: that's why i didn't; i'll shorten the name of the defined value i suppose.
@crudelios is there a style checker in the CI pipeline? in prior projects i've always had to deal with line length

Reply:
Well, the names do tend to be verbose, but I always felt it makes the code clearer.

>  is there a style checker in the CI pipeline?

Not in the CI pipeline, but we have a very strict style checker called @bvschaik :wink: But adding an automated style checker would be a good idea... Let's see if I can find one for C!

Reply:
> Let's see if I can find one for C!
what have i done... :laughing: 

another reason i did this is because of line 335 -  
```
int y_offset = FILLER_Y_OFFSET;
```
y_offset doesnt change throughout draw_extra_info_panel.   ill remove y_offset, if no-one has any objections

Reply:
I certainly agree on adding an automatic style checker. If I can be automated, please do 😉 

> what have i done... 😆

Just an extra new line where one shouldn't be 😉 
I'm quite strict on brace positions, spacing and other formatting things; @crudelios knows from my comments on previous pull requests 😆 

> y_offset doesnt change throughout draw_extra_info_panel. ill remove y_offset, if no-one has any objections

No objection here.